starboard/tvos/shared/media_is_supported.mm

```diff
@@ -21,9 +21,14 @@
 const char kWidevineL3SystemName[] = "com.youtube.widevine.l3";
 const char kWidevineForceHdcpSystemName[] = "com.youtube.widevine.forcehdcp";
 
-bool SbMediaIsSupported(SbMediaVideoCodec video_codec,
-                        SbMediaAudioCodec audio_codec,
-                        const char* key_system) {
+namespace starboard {
+namespace shared {
+namespace starboard {
+namespace media {
```

Comment:
You should flatten this into `namespace starboard::shared::starboard::media {` (see e.g. #5908).